packages/core/package.json

```diff
@@ -19,10 +19,13 @@
 	},
 	"devDependencies": {
 		"@types/node": "^20.12.8",
-		"tsup": "^8.0.2",
+		"@rttist/typegen": "^0.1.0-proto.13",
+		"rttist": "^1.0.0-beta.8",
 		"typescript": "^5.4.5",
 		"esbuild-plugin-solid": "^0.5.0",
-		"component-register": "^0.8.3"
+		"component-register": "^0.8.3",
+		"vite": "^5.2.11",
+		"vite-plugin-dts": "^3.9.1"
```

Comment:
might need vite-plugin-solid listed here too with pnpm

Reply:
It's interesting it even works without it... PNPN prevents using packages not listed in package.json.

Reply:
Yeah I was confused too.